### ISSUE TITLE
You can not update your email address properly

### ISSUE THREAD

**Issue by ghost** [python/pythondotorg#813]
The problem persists in email **updating** process; **use case**: when an user registers an account with his email address (i.e. a@mail.ltd) and next he changes his email with a new (i.e. b@mail.ltd) the previous email (a@mail.ltd) remains linked to the first account registration. 
- The **recovery password** will always work with first registered email (a@mail.ltd).
- If the user wants to create a **new account** with the **first registered email** (now changed with b@mail.ltd), he **can't** because the email results already used.

Cheers!
M

**Comment by sabareeshkk** [comment]
yes. the email id is changed!!!..the email id field have to be removed from the edit profile page.
registered email id is unique so we doesn't  have the option to update it right?.
if want to update the email then we have to check if it available or not. then we can update it.

**Comment by ghost** [comment]
Is there any update about this issue?

**Comment by sabareeshkk** [comment]
no.i can remove it from the edit page .but  could it be a valid solution?

**Comment by root-11** [comment]
I've got this problem right now. I've registered with email A, forgotten the username, try to register with email B and than change the email back to A. Fail.

So why is this necessary? Because (A) I can't sign in with my email, and (B) I can't retrieve my username from the database using my email.
